bundles/org.openhab.binding.alarmdecoder/README.md

```diff
@@ -12,7 +12,7 @@ There are several versions of the adapter available:
 This binding allows openHAB to access the state of wired or wireless contacts and motion detectors connected to supported alarm panels, as well as the state of attached keypads and the messages send to attached LRR devices.
 Support is also available for sending keypad commands, including special/programmable keys supported by your panel.
 
-For those upgrading from the OH1 version of the binding, the [original OH1 README](https://www.openhab.org/v2.5/addons/bindings/alarmdecoder1/) file is available for reference.
+For those upgrading from the OH1 version of the binding, the [original OH1 README](https://github.com/openhab/openhab1-addons/blob/main/bundles/binding/org.openhab.binding.alarmdecoder/README.md) file is available for reference.
```

Comment:
I'd prefer to remove this completely, since this binding was already in place in v2 and people migrating from v1 bindings should know for ALL their bindings how they had set up their v1 version.

Reply:
Ok. I've removed it. But I would feel better about it if there was a way to easily access the V1 binding docs from openhab.org somehow. As I mentioned in https://github.com/openhab/website/issues/275 yesterday, there doesn't seem to be. They are available on github, of course, but I suspect most users wouldn't be able to find them there.